### ISSUE TITLE
dataclasses module not found

### ISSUE THREAD

**Issue by crigaut** [rr-/screeninfo#19]
I installed the module via pip and when I run `from screeninfo import get_monitors` I get this error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/dist-packages/screeninfo/__init__.py", line 1, in <module>
    from .common import Enumerator, Monitor
  File "/usr/local/lib/python3.6/dist-packages/screeninfo/common.py", line 3, in <module>
    from dataclasses import dataclass
ModuleNotFoundError: No module named 'dataclasses'
```
Is there a way to solve that?

**Comment by juanjavatus** [comment]
Hi,
I've found dataclasses is new in python 3.7. You have a backport to 3.6:
pip install dataclasses
regards

**Comment by rr-** [comment]
Thanks, released 0.4.1 with dataclasses as a dependency.
